src/components/App/index.tsx

```diff
@@ -43,7 +42,7 @@ interface SourceImage {
   preprocessed?: ImageData;
 }
 
-interface EncodedImage {
+export interface EncodedImage {
```

Comment:
yup. I think without gzip it's no longer necessary, but if it is I can hoist it.

Reply:
So I removed it and replaced the `Options` usage with its own type, since it only cares about 2 properties. Not sure if that's going against TS idioms though.